packages/grafana-ui/src/components/ClipboardButton/ClipboardButton.tsx

```diff
@@ -83,3 +105,24 @@ const copyText = async (text: string, buttonRef: React.MutableRefObject<HTMLButt
     input.remove();
   }
 };
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    button: css({
+      position: 'relative',
+    }),
+    successButton: css({
+      '> *': css({
+        visibility: 'hidden',
+      }),
+    }),
+    successOverlay: css({
+      position: 'absolute',
+      top: 0,
+      bottom: 0,
+      right: 0,
+      left: 0,
+      visibility: 'visible', // re-visible the overlay
+    }),
```

Comment:
I want to change the button to show _just_ a check, but not change the size. This was the best I could come up with, but it's pretty icky.

I was hesitant to add this functionality directly into Button

Reply:
The only other way I can think of is if you check the size of the Button after first render and then just keep it to that with any change, but I think that would be even _more_ hacky than doing an overlay...